ios/FluentUI/Vnext/Avatar/AvatarGroup.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public enum MSFAvatarGroupStyle: Int, CaseIterable {
+    case stack
+    case pile
+}
+
+/// Properties that make up AvatarGroup content
+@objc public class MSFAvatarGroupState: NSObject, ObservableObject {
+    @objc @Published public var avatars: [MSFAvatarStateImpl] = []
+}
+
+/// View that represents the AvatarGroup
+public struct AvatarGroup: View {
+    @Environment(\.theme) var theme: FluentUIStyle
```

Comment:
We need to have the window provider environment value here. (#551 for reference)

Reply:
Have you added the designTokens modifier to AvatarGroup?
Without it the AvatarGroup overriden tokens won't be applied.

Can you make sure you test it by setting a property based on the window theme to ensure it changes accordingly?